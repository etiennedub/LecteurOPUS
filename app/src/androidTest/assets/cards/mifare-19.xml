<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">044E25E7</List><List index="1">0A674980</List><List index="2">A448E000</List><List index="3">00000003</List><List index="4">00000000</List><List index="5">3129C520</List><List index="6">6F540000</List><List index="7">3ABB886F</List><List index="8">01C2AA3E</List><List index="9">A3E1030C</List><List index="10">750038D4</List><List index="11">20281B52</List><List index="12">01C2A82A</List><List index="13">82A6050A</List><List index="14">6800B8D4</List><List index="15">2028D65B</List></Pages></Mifare>