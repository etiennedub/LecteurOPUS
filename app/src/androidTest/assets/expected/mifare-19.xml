<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="28/09/2016" Time="21:51" Transfert="false" OperatorId="20" BusId="16" OperatorName="RTC" BusName="LEBUS 28" /><List index="1" Date="28/09/2016" Time="17:25" Transfert="false" OperatorId="20" BusId="96" OperatorName="RTC" BusName="METROBUS 801" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>